setup.py

```diff
@@ -33,6 +33,7 @@
         "Source Code": "https://github.com/torchsynth/torchsynth/",
     },
     packages=find_packages(exclude=("tests", "examples")),
+    package_data={"torchsynth": ["nebulae/voice/*.json"]},
```

Comment:
this is better I think -- we will need to add the folders here for new synth nebulas when we add them (e.g. FM) -- but for now I think this is the cleanest way to add json files to the package

Reply:
nice